astropy/time/formats.py

```diff
@@ -573,6 +573,69 @@ def value(self):
 
         return iterator.operands[-1]
 
+    def to_value(self, timezone=None):
+        """
+        Convert to (potentially timezone-aware) `~datetime.datetime` object.
+
+        If ``timezone`` is not `None`, return a timezone-aware datetime
+        object.
+
+        Parameters
+        ----------
+        timezone : {`~datetime.tzinfo`, None} (optional)
+            If not `None`, return timezone-aware datetime.
+
+        Returns
+        -------
+        `~datetime.datetime`
+            If ``timezone`` is not `None`, output will be timezone-aware.
+        """
+        if timezone is not None:
+            if self._scale != 'utc':
+                raise ScaleValueError("Scale is {}, must be UTC."
```

Comment:
Use `"scale is {}, must be 'utc' when timezone is supplied."`

BTW, I've started following the convention used by Python itself of starting exceptions with lower-case, e.g. 
```
In [27]: 1/0
ZeroDivisionError: integer division or modulo by zero
```